cloudflare/resource_cloudflare_byo_ip_prefix_test.go

```diff
@@ -0,0 +1,46 @@
+package cloudflare
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestAccCloudflareBYOIPPrefix(t *testing.T) {
+	t.Parallel()
+	prefixID := os.Getenv("CLOUDFLARE_BYO_IP_PREFIX_ID")
+
+	rnd := generateRandomResourceName()
+	name := fmt.Sprintf("cloudflare_byo_ip_prefix.%s", rnd)
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheckBYOIPPrefix(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccCheckCloudflareBYOIPPrefixConfig(prefixID, "BYOIP Prefix Description old", rnd),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(
+						name, "description", "BYOIP Prefix Description old"),
+				),
+			},
+			{
+				Config: testAccCheckCloudflareBYOIPPrefixConfig(prefixID, "BYOIP Prefix Description new", rnd),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(
+						name, "description", "BYOIP Prefix Description new"),
+				),
+			},
+		},
+	})
+}
+
+func testAccCheckCloudflareBYOIPPrefixConfig(prefixID, description, name string) string {
+	return fmt.Sprintf(`
+				resource "cloudflare_byo_ip_prefix" "%[3]s" {
+					prefix_id = "%[1]s"    
+					description = "%[2]s"
+				}`, prefixID, description, name)
```

Comment:
Can we format this JSON to not have really big indents?

Reply:
I removed the indentation, I hope that meant with formatting? :-)